RocketChat/Rocket.Chat
# [IMPROVE] Replace remaing discussion creation modals with React modal.

## Proposed changes (including videos or screenshots)


### before
![image](https://user-images.githubusercontent.com/27704687/123840524-cbe72b80-d8e4-11eb-9ddb-23a9f9d90aac.png)

### after
![image](https://user-images.githubusercontent.com/27704687/123840219-74e15680-d8e4-11eb-95aa-00a990ffe0e7.png)




## Issue(s)


## Steps to test or reproduce

Try to create a discution from a message and from the message box (message editor plus sign)
When creating from a message, the title of the discution should already be filled with part of the message.
In both cases, the "parent room' field should be disabled, and the current room name should be displayed there.
## Further comments
